src/electionguard/ballot.py

```diff
@@ -172,7 +171,6 @@ class CiphertextBallotSelection(
     proof: Optional[DisjunctiveChaumPedersenProof] = field(default=None)
     """The proof that demonstrates the selection is an encryption of 0 or 1, and was encrypted using the `nonce`"""
 
-    # TODO: ISSUE #35: encrypt/decrypt
     extended_data: Optional[ElGamalCiphertext] = field(default=None)
```

Comment:
Is a different issue for changing this to HashedElgamalCiphertext instead of the ElGamalCiphertext?